packages/components/src/popover/detect-outside.js

```diff
@@ -1,24 +1,21 @@
 /**
- * External dependencies
+ * WordPress dependencies
  */
-import clickOutside from 'react-click-outside';
```

Comment:
Unrelated question to this PR. Can we also refactor `Modal` component to stop using `react-click-outside`? :)

Reply:
> Unrelated question to this PR. Can we also refactor `Modal` component to stop using `react-click-outside`? :)

I would like to, yes.

( https://github.com/WordPress/gutenberg/pull/6261#discussion_r191758392 😞 )

Reply:
There is still hope then :)

Reply:
https://unpkg.com/react-click-outside@3.0.1/index.js

Well, it looks like `react-click-outside` is a simplified version of our own HOC :)

Reply:
I opened a follow-up to address it – #16878.